Add posibility to disable noblanks method in Signer initialization

bpietraga opened the pull request:
Thank you for amazing gem!

In my work i discovered that changing of xml structure such as  Nokogiri `noblanks` needed for canonicalization of XML in default behavior breaks ability to validate in one of endpoints that I'm working with.

I've added option for disabling `noblanks` in Signer initialization with optional argument `should_trim: false`, default gem behavior stays the same and all previous tests pass except on ruby 2.0.

ebeigarts commented on lib/signer.rb:
```diff
@@ -14,8 +14,8 @@ class Signer
 
-  def initialize(document)
-    self.document = Nokogiri::XML(document.to_s, &:noblanks)
+  def initialize(document, should_trim: true)
```
I think it would be better to stick with nokogiri naming:

```
def initialize(document, noblanks: true)
  self.document = Nokogiri::XML(document.to_s) do |config|
     config.noblanks if noblanks
  end
  ...
```

bpietraga replied:
I will fix it

bpietraga replied:
I have changed everything to nokogiri naming. It looks better now 😃

bpietraga commented:
@ebeigarts I have added fix for ruby version 2.0.0 + Nokogiri dependency issue which caused test to fail. Newer versions of Nokogiri require ruby at least 2.1.

With this fix legacy ruby 2.0 users can install working Signer. Please let me know if there Is a need for changing something.

ebeigarts commented on signer.gemspec:
```diff
@@ -19,5 +19,9 @@ Gem::Specification.new do |gem|
+    gem.add_runtime_dependency "nokogiri", ">= 1.5.1"
+  else
+    gem.add_runtime_dependency "nokogiri", "~>1.6.8"
+  end
```
This won't work, because gemspec is evaluated when building the `.gem`. Let's drop `2.0`. Can you add `spec.required_ruby_version = '>= 2.1.0'` and remove `2.0` from travis.yml?

bpietraga commented:
@ebeigarts Thank you for reply!
I've added required_ruby_version `gem.required_ruby_version = '>= 2.1.0'`. Also changed `'rake'` and `'rspec'` to single quotes just to keep it tidy.
Plus ruby 2.0 is removed from travis.yml

ebeigarts commented:
Thanks, released in v1.5.0
